### PR TITLE
fix bug:  fix actiontimeline can not step to last frame when loop play

### PR DESCRIPTION
**Pull request by geron-cn**
fix bug:  fix actiontimeline can not step to last frame when loop play

### CODE REVIEW

**Comment by zilongshanren** [comment]
@chengstory  Could you confirm this PR?
Thanks.

**Comment by geron-cn** [comment]
@2youyouo2  Could you check this out?

**Comment by pandamicro** [comment]
Related issue: https://github.com/cocos2d/cocos2d-x/issues/12244

**Comment by pandamicro** [comment]
@jicaixin Can you help to review this Pull Request ? Does it solve your problem ?

**Comment by pandamicro** [comment]
@VisualSJ It should be merged to JS

File: cocos/editor-support/cocostudio/ActionTimeline/CCActionTimeline.cpp [hunk 1]
```diff
@@ -199,23 +199,28 @@ void ActionTimeline::step(float delta)
     }
 
     _time += delta * _timeSpeed;
-    
-    if(_time < _endFrame * _frameInternal)
+    const float endtoffset = _time - _endFrame * _frameInternal;
```
Comment (VisualSJ) [inline cocos/editor-support/cocostudio/ActionTimeline/CCActionTimeline.cpp:202]: The parameter name is very strange

**Comment by pandamicro** [comment]
Reviewed by @VisualSJ

**Pull request merged**